dali/pipeline/operator/builtin/external_source.h

```diff
@@ -237,56 +236,65 @@ class ExternalSource : public Operator<Backend> {
   }
 
   template<typename SrcBackend, template<typename> class SourceDataType>
-  inline std::enable_if_t<!std::is_same<SrcBackend, Backend>::value, bool>
-  ShareData(const SourceDataType<SrcBackend> &t, cudaStream_t /*stream = 0*/) {
-    DALI_FAIL("No_copy is supported only for the same data source device type as operator.");
-    return false;
+  inline std::enable_if_t<!std::is_same<SrcBackend, Backend>::value, void>
```

Comment:
Small nitpick: you don't need that `void`, it's there by default.

Reply:
Same in the two below.

Reply:
Done